ncm-systemd/src/main/perl/Systemd/Service.pm

```diff
@@ -619,16 +619,26 @@ sub change
         }
     }
 
-    # TODO: same TODOs as with states
-    foreach my $act (sort keys %$acts) {
-        my @units = @{$acts->{$act}};
-
-        # TODO: process units wrt dependencies?
-        if (@units) {
-            # TODO: trap exitcode and stop any further processing in case of error?
-            # CAF::Process logger is sufficient
-            systemctl_command_units($self, $change_activation->{$act}, @units);
+    my $multi_user_active = systemctl_is_active($self, "multi-user.target");
```

Comment:
can you make a readonly/constant for the 'multi-user.target' and use that instead of the magic string

my only remark is if there would be a reason not to do this, and thus to make this configurable to skip. i can't think of any, so i'm just asking.